### DISCUSSION TITLE
Issue with AWS

### DISCUSSION THREAD

**Discussion by aitoehigie** [geetfun/deployed#2] [Q&A]
I am trying to use this gem to run an app on AWS but I keep getting this error:
Host key verification failed.
ERROR (Net::SSH::Proxy::ConnectError): Exception while executing on host [redacted]: command failed

Can anyone advise me on how to set up ssh in the config/deploy.yml file?

**Comment by geetfun** [comment]
@aitoehigie Not sure if you solved this already. With AWS, my best guess is either there's some firewall restriction or the SSH key doesn't match. 

Check to see you can SSH into your server (eg. `ssh root@x.x.x.x`).

Will close this discussion as this isn't directly related to this gem or `kamal` per se.
